fighthealthinsurance/models.py

```
@@ -825,3 +828,128 @@
     error = models.CharField(max_length=255, null=True, blank=True)
+
+
+class PriorAuthRequest(ExportModelOperationsMixin("PriorAuthRequest"), models.Model):  # type: ignore
+    """
+    Stores information about a prior authorization request.
+    Used to track the status of the request and store the questions and answers.
+    """
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    creator_professional_user = models.ForeignKey(
+        ProfessionalUser,
```

Ruff (0.8.2)

`ProfessionalUser` may be undefined, or defined from star imports

(F405)